action/receipt.go

```diff
@@ -119,16 +115,34 @@ func (receipt *Receipt) Hash() hash.Hash256 {
 	return hash.Hash256b(data)
 }
 
+// Logs returns the list of transaction logs stored in receipt
+func (receipt *Receipt) Logs() []*Log {
+	return receipt.logs
+}
+
 // AddLogs add log to receipt and filter out nil log.
-func (receipt *Receipt) AddLogs(logs ...*Log) {
-	if receipt.Logs == nil {
-		receipt.Logs = make([]*Log, 0, len(logs))
```

Comment:
should keep this?
saw places construct r = &action.Receipt{ ... } with logs = nil